src/instructlab/train/lora_mlx/make_data.py

```diff
@@ -31,7 +31,7 @@ def make_data(data_dir: str, is_shiv: bool = False):
             data_new = []
             for obj in data:
                 obj_new = {
-                    "system": get_sysprompt(),
+                    "system": get_sysprompt("default"),
```

Comment:
I feel bad you had to wire up lora_mlx, linux_train, and test.py given that we should probably deprecate those commands 😓 


@nathan-weinberg @alinaryan lets scope work to deprecate those workflows so we don't need to deal with wiring things like this up again in those placed, WDYT?

Reply:
yeah, this code is a PITA